### ISSUE TITLE
 Using NetPGA-1G-CML in PCIe mode

### ISSUE THREAD

**Issue by eselahe** [cmlab/NetFPGA-1G-CML#3]
Hi,
I want to use NetFPGA in PCIe mode. I install the driver by "make" command which located in "/projects/reference_nic_nf1_cml/sw/host/driver", successfully and nf10.ko file is created. then I reboot my PC. After restarting the computer, I run the command [lspci], but Xilinx card not visible. My motherboard model is ASUS P8Z68-V LE. 
please help me.

**Comment by davidva-cml** [comment]
Hello,

Please be sure you have configured the FPGA with the project bitstream, 
located in the "bitfiles" directory, before rebooting your machine and 
running lspci.  If you pull the latest commit, running "make download" 
from /projects/reference_nic_nf1_cml/ will configure the FPGA with the 
project bitstream.

David A. Van Arnem
Development Engineer
Computer Measurement Laboratory

**Comment by eselahe** [comment]
My problem solved. Thanks.

**Comment by jmeador** [comment]
Hi -
How did you fix your problem?
This will help other people in the future when they read this thread.
Thanks!

**Comment by eselahe** [comment]
Hi,
Ok, I was wanting test the PCIe interface based on the answer to below
question in NetFPGA-1G-CML-FAQ page:
Q: Does the PCIe mode require a specific PCIe driver?
But in answer to this question, it has not been explained that it is
needed the bitfile of reference_nic_nf1_cml project for configuration
of the FPGA. also I was using NetFPGA-10G-live package in devel branch
which was not last update. I downloaded last update of this package.
But the "make download" rule didn't workd for transfering bitfile to
PCB. So I downloaded this packge from master branch, I maked cores.
After that I ran "make downlod" rule for the reference_nic_nf1_cml
project. I programmed the board successfully and rebooted my PC and
then I ran "lspci" command, the xilinx card added to my PC. But I have
another problem when I test the NetFPGA as NIC. I will ask it in a new
issue.

Best regards
